griptape/config/lm_studio_structure_config.py

```diff
@@ -0,0 +1,54 @@
+from attrs import Factory, define, field
+
+from griptape.config import StructureConfig
+from griptape.drivers import (
+    BaseEmbeddingDriver,
+    BaseImageGenerationDriver,
+    BaseImageQueryDriver,
+    BasePromptDriver,
+    BaseVectorStoreDriver,
+    LocalVectorStoreDriver,
+    OpenAiChatPromptDriver,
+    OpenAiEmbeddingDriver,
+    OpenAiImageGenerationDriver,
+    BaseTextToSpeechDriver,
+    OpenAiTextToSpeechDriver,
+    BaseAudioTranscriptionDriver,
+    OpenAiAudioTranscriptionDriver,
+    OpenAiImageQueryDriver,
+)
+
+
+@define
+class OpenAiStructureConfig(StructureConfig):
```

Comment:
```suggestion
class LmStudioStructureConfig(StructureConfig):
```

Reply:
@vachillo this was an accidental check in. I'm actually not sure what a Structure Config would look like for LM Studio since there are no default models.

Suggestions?

Reply:
ah i see. i would say either make it all the same as the OpenAi default models or just remove it

Reply:
basically just 
```@define
class OpenAiStructureConfig(OpenAiStructureConfig):
    pass
```
i guess right

Reply:
Yeah removed for now.